src/hooks/useTabletMode.ts

```diff
@@ -0,0 +1,19 @@
+import { useState, useEffect } from 'react';
+import { TABLET_TRESHOLD } from '@/components/variables';
+
+export const useTabletMode = () => {
+  const [width, setWidth] = useState<number>(window.innerWidth);
+
+  const handleWindowSizeChange = () => setWidth(window.innerWidth);
+
+  useEffect(() => {
+    window.addEventListener('resize', handleWindowSizeChange);
+    return () => {
+      window.removeEventListener('resize', handleWindowSizeChange);
+    };
+  }, []);
+
+  const isTablet = width <= TABLET_TRESHOLD;
+
+  return isTablet;
+};
```

Comment:
Instead of having two almost identical (useTabletMode, useMobileMode) hooks, I would preferred to refactor the useMobileMode to take an argument of the widht treshhold, or perhaps even just a enum to descripbe what size are we checking

ie.

```typescript
 const isTabletMode = useScreenSize('tablet')
 ```

Reply:
Or even maybe better, to extend it to give a record of modes, no argument needed

```typescript
const {isDesktop, isTablet, isMobile} = useScreen()
```

Not sure if we want `isDesktop`, since it would be default

Reply:
Yes, I'd like to see this too!

Reply:
Fixed this